app/models/apple/config.rb

```diff
@@ -55,14 +62,28 @@ def self.find_or_build_private_feed(podcast)
     end
 
     def self.build_apple_config(podcast, key)
-      ac = Apple::Config.new
+      if has_apple_config?(podcast)
+        Rails.logger.error("Found existing apple config for #{podcast.title}!")
+        Rails.logger.error("Do you want to continue? (y/N)")
+        raise "Stopping build_apple_config" if $stdin.gets.chomp.downcase != "y"
+      end
```

Comment:
It's possible we can completely automate this at some point, and drop these checks.